docs/Developer's Guide.md

```diff
@@ -0,0 +1,322 @@
+# Developer's Guide
+
+This guide walks you through the basics of developing the Gordon 360 frontend, all the way from setting up an initial development environment to deploying changes to production.
+
+# Contents
+
+- [Getting Started](#getting-started)
+- [Connecting to the Backend](#connecting-to-the-backend)
+  - [Connecting to Remote Backend via SSH](#connecting-to-remote-backend-via-ssh)
+- [Code Style](#code-style)
+- [Project File Organization](#file-organization)
+  - [Components](#components)
+  - [Contexts and Hooks](#contexts-and-hooks)
+  - [Services](#services)
+  - [Views](#views)
+- [Libraries](#libraries)
+- [Continuous Integration and Deployment](#continuous-integration-and-deployment)
+
+# Getting Started
+
+Here are the steps to setupthe frontend for development:
+
+1. Clone this repository to the machine that you will develop on.
+
+1. Ensure that [NodeJS](https://nodejs.org/en/) is installed on that machine.
+
+   From a terminal, run this command:
+
+   ```
+   node -v
+   ```
+
+   If it returns a version number, then NodeJS is installed. Proceed to the next step.
+
+   If it doesn't return a number, you will need to install NodeJS. There are two ways to do this:
+
+   1. Download and run the installer from [the NodeJS Website](https://nodejs.org/en/download/)
+   1. Use a tool that manages Node installations, such as
+      - For any OS: [Fast Node Manager](https://github.com/Schniz/fnm)
+      - For MacOS/Linux: [Node Version Manager](https://github.com/nvm-sh/nvm)
+      - For Windows: [NVM for Windows](https://github.com/coreybutler/nvm-windows)
+
+1. Install dependencies
+
+   From a terminal, go to the directory where you cloned the project (e.g. the `gordon-360-ui` folder) and run the following command:
+
+   ```
+   npm install
+   ```
+
+   This will install the frontend's dependencies. It may take a long time the first time you run it.
+
+1. Start the project
+
+   From a terminal in the project directory, run this command:
+
+   ```
+   npm run start
```

Comment:
```suggestion
   npm start
```
Can't this be simplified to just the above?

Reply:
Sure, but they need to do `npm run ____` for all the other commands in `package.json`, so I figured for consistency's sake I'd leave it as `npm run start`.